js/views/currencyListVw.js

```diff
@@ -29,11 +29,12 @@ module.exports = Backbone.View.extend({
     orderedCurrencies.unshift({code: "BTC", currency: "Bitcoin", currencyUnits: "4"});
     this.chooseCurrencies = new chooseCurrenciesCollection(orderedCurrencies);
     this.subViews = [];
+
     //use default currency to return list of supported currencies
     getBTPrice("USD", function (btAve, currencyList) {
-      "use strict";
```

Comment:
Somehow the getBTPrice function is being called twice inside initialize, even though initialize only gets called once. This causes the currency list to be rendered twice, which adds it to the DOM twice.